app/build.gradle

```diff
@@ -33,7 +33,7 @@ android {
 
   defaultConfig {
     applicationId "com.mapzen.erasermap"
-    minSdkVersion 15
+    minSdkVersion 17
```

Comment:
Why the change for `minSdkVersion`?

Its probably fine to drop support for API 15 (2%) but there are still 7.2% of devices on API 16 currently.